webapp/src/views/projects/languages/tableStyles.ts

```diff
@@ -0,0 +1,39 @@
+import { makeStyles } from '@material-ui/core';
```

Comment:
It's maybe better to keep the styles with components, but split the table to more components, isn't it?